server/views/components/event-promo/event-promo.njk

```diff
@@ -24,13 +24,19 @@
     </div>
 
     {% for dateRange in model.when.toJS() %}
+      {% set formattedDateRange = dateRange.start | formatAndDedupeOnDate(dateRange.end) %}
       <div class="{{ {s:1} | spacingClasses({margin: ['bottom']}) }}">
         <div class="{{ data.dateFont | fontClasses }}">
-          {{ dateRange.start | formatAndDedupeOnDate(dateRange.end) | join(' - ') }}
-        </div>
-        <div class="{{ data.timeFont | fontClasses }}">
-          {{ dateRange.start | formatAndDedupeOnTime(dateRange.end) | join(' - ') }}
+          {{ formattedDateRange | join(' - ') }}
```

Comment:
Could this join with an en-dash `–` rather than a hyphen `-`? That's the format we've used on the exhibition page and (I think) is the standard for date ranges.

Reply:
Could you explain from a users perspective what difference it makes?

Reply:
I did do it - just really interested?